swift-package/Resources/ContentScopeScripts.swift

```diff
@@ -0,0 +1,5 @@
+import Foundation
+
+public struct ContentScopeScripts {
+    public static var bundle: Bundle = .module
```

Comment:
I would suggest adding a helper here for getting the content scope script's URL within its bundle. This would contain the details of how to access the resource (e.g. subfolders and filenames) within the package.